content/articles/2018-02-tidyr-0-8-0.Rmd

```diff
@@ -0,0 +1,51 @@
+---
+title: tidyr 0.8.0
+author: Hadley Wickham
+date: '2018-02-13'
+slug: tidyr-0-8-0
+categories:
+  - package
+tags: []
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+library(tidyr)
+library(tibble)
+```
+
+We are pleased to announce that tidyr 0.8.0 is now available on CRAN. tidyr makes it easy to “tidy” your data, storing it in a consistent form so that it’s easy to manipulate, visualise and model. Tidy data has a simple convention: put variables in the columns and observations in the rows. You can learn more about it in the [tidy data vignette](http://tidyr.tidyverse.org/articles/tidy-data.html). Install it with:
+
+```{r, eval = FALSE}
+install.packages("tidyr")
+```
+
+This release mainly contains a bumper crop of small bug fixes and minor improvements, and a considerably increase in test coverage (84% to 99%). For the full details, see the [release notes](https://github.com/tidyverse/tidyr/releases/tag/v0.8.0). Here we'll highlight an important bug fix that might change existing code, and one new feature you might like to try out.
```

Comment:
Swap one of the "mights" to a could, or just rephrase the last part of the sentence "…and a new feature to try out." (Hey, if they don't want to, I'm sure they won't!) 😉